packages/mask/src/plugins/RedPacket/SNSAdaptor/RedPacketInHistoryList.tsx

```diff
@@ -214,11 +214,12 @@ export function RedPacketInHistoryList(props: RedPacketInHistoryListProps) {
     const rpid = receipt?.rpid ?? ''
     const creation_time = receipt?.creation_time ?? 0
 
+    history.rpid = rpid
```

Comment:
Don't change props!!!

Reply:
He's supposed to patch `history` and reuse below.
Fixed in https://github.com/DimensionDev/Maskbook/pull/6795